go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/bufbuild/buf
 
-go 1.21.0
-
-toolchain go1.23.0
+go 1.22
```

Comment:
We need to figure out if this is OK

Reply:
This should be fine if we're requiring Go 1.22+:

`go`: minimum version required for the language features in the module (that are not behind a build flag)
`toolchain`: recommended stdlib and compiler to use when building the module (ignored transitively), must be greater than or equal to the `go` directive (and elided if equal)

It's dense, but the [docs](https://go.dev/doc/toolchain) on toolchains are helpful.